addons/dark-www/addon.json

```
@@ -2445,62 +2445,62 @@
       }
     },
     {
-      "name": "Scratch 2.0 (2015)",
-      "id": "scratch-oldWwwBlue",
-      "description": "The colors of the home page between 2015 and 2018",
+      "name": "Scratch's default colors (blue)",
+      "id": "scratch-blue",
+      "description": "The colors used by Scratch between 2019 and 2023",
       "values": {
         "page": "#fcfcfc",
-        "navbar": "#25aff4",
+        "navbar": "#4d97ff",
         "box": "#ffffff",
         "gray": "#f2f2f2",
-        "blue": "#e6f3fa",
+        "blue": "#e9f1fc",
         "input": "#fafafa",
-        "button": "#25aff4",
-        "link": "#25aff4",
+        "button": "#4d97ff",
+        "link": "#4d97ff",
         "footer": "#f2f2f2",
         "border": "#0000001a",
+        "messageIndicatorColor": "#ffab1a",
+        "messageIndicatorOnMessagesPage": "#ffab1a",
         "darkBanners": "off",
         "darkForumCode": false
       }
     },
     {
-      "name": "Scratch's default colors",
-      "id": "scratch",
-      "description": "The colors normally used by Scratch",
+      "name": "Scratch's default colors (light blue)",
+      "id": "scratch-oldWwwBlue",
+      "description": "The colors of the home page between 2015 and 2018",
       "values": {
         "page": "#fcfcfc",
-        "navbar": "#855cd6",
+        "navbar": "#25aff4",
         "box": "#ffffff",
         "gray": "#f2f2f2",
-        "blue": "#e9f1fc",
+        "blue": "#e6f3fa",
         "input": "#fafafa",
-        "button": "#855cd6",
-        "link": "#855cd6",
+        "button": "#25aff4",
+        "link": "#25aff4",
         "footer": "#f2f2f2",
         "border": "#0000001a",
-        "messageIndicatorColor": "#ffab1a",
-        "messageIndicatorOnMessagesPage": "#cc4400",
         "darkBanners": "off",
         "darkForumCode": false
       }
     },
     {
-      "name": "Scratch's default colors (blue)",
-      "id": "scratch-blue",
-      "description": "The colors originally used by Scratch",
+      "name": "Scratch's default colors",
```

It looks weird if the last option doesn't have a note in brackets.
```suggestion
      "name": "Scratch's default colors (purple)",
```